### PR TITLE
Add ephemeral container to a running pod

### LINKED ISSUE #30: Add function to start an ephemeral container to a pod
In order to support https://github.com/grafana/k6-chaos/issues/7 It is necessary to add the option to start an ephemeral container in a pod.

### PR DESCRIPTION
**Pull request by pablochacin**
Add function to Pods for creating an ephemeral container in a running pod.
Also added a function for creating a pod to facilitate testing.

Note: tested with K8s cluster 1.23.x. Previous versions may not work or may need enabling ephemeral containers feature gate

closes #30

### CODE REVIEW

**Review by javaducky — CHANGES_REQUESTED** [review]

**Comment by javaducky** [comment]
Confirmed create and add with Kube v1.22.7 (`feature-gates=EphemeralContainers=true`).

File: pkg/pods/pods.go [hunk 1]
```diff
@@ -17,12 +20,27 @@ func New(client *kubernetes.Clientset, metaOptions metav1.ListOptions, ctx conte
 	}
 }
 
+type ContainerOptions struct {
```
[the remark this answers is not in the document]
Comment (pablochacin) [reply pkg/pods/pods.go:24]: Hi @javaducky I added some comments, but I'm not sure what conventions is the project following in this regard. Could you please point me to some examples I can use as a reference?

[the remark this answers is not in the document]
Comment (javaducky) [reply pkg/pods/pods.go:24]: Just standard documentation as described with [godoc](https://go.dev/blog/godoc) is fine. No need to worry about all the methods/types; just the new ones that you're adding.

We haven't been good at requiring this, but [xk6-sql](https://github.com/grafana/xk6-sql/blob/master/sql.go) is an example where I added documentation. You can always use `golint ./...` to check your progress.

[the remark this answers is not in the document]
Comment (pablochacin) [reply pkg/pods/pods.go:24]: Added documentation.

**Review by javaducky — APPROVED** [review]

**Comment by javaducky** [comment]
LGTM! Thank you @pablochacin .

**Pull request merged**